preprocessing-service/preprocess.py

```diff
@@ -46,15 +47,14 @@ async def doc_generator(df):
             if not k in doc_keywords:
                 doc_dict["doc"][k] = doc_dict[k]
                 del doc_dict[k]
-        doc_dict["doc"]["drain_control_plane_template_matched"] = ""
-        doc_dict["doc"]["anomaly_level"] = "Normal"
         yield doc_dict
 
 
 async def consume_logs(mask_logs_queue):
     async def subscribe_handler(msg):
+
         payload_data = msg.data.decode()
-        await mask_logs_queue.put(pd.read_json(StringIO(payload_data), dtype={"_id": object, "cluster_id": str}))
+        await mask_logs_queue.put(json.loads(payload_data))
```

Comment:
this line can't be changed because this was introduced to support the opni support bundle

Reply:
it has to be `await mask_logs_queue.put(pd.read_json(StringIO(payload_data), dtype={"_id": object, "cluster_id": str}))`

Reply:
@sanjay920  why can't we remove it though? if the support bundle can adapt to our latest pipeline, this shouldn't be a problem